apps/site/lib/site_web/templates/trip_plan/_to_from_inputs.html.eex

```diff
@@ -31,11 +31,16 @@
     <div id="trip-plan__location-error--from" role="alert" aria-live="assertive" tabindex="0" class="c-trip-plan-widget__location-error location-error"><%= @from_error %></div>
   </div>
 
+  <div id="trip-plan-reverse-control" class="c-trip-plan-widget__reverse-control">
```

Comment:
![image](https://user-images.githubusercontent.com/526017/170776069-9f24c735-6dea-4436-907a-8998641a665f.png)

When the error container is visible, the arrows seem to shift quite a bit.

Reply:
Did a hot fix by putting a display none on the error container.

Reply:
I'll approve this for now and try to fix the shifting in the PR I'm currently working on w/ the error handling.